nixos/modules/services/web-apps/movim.nix

```diff
@@ -621,7 +621,7 @@ in
             DB_DRIVER =
               {
                 "postgresql" = "pgsql";
-                "mysql" = "mysql";
+                "mariadb" = "mysql";
```

Comment:
I haven't looked at the actual code changes to drop mysql support, but I'm surprised the value of this configuration setting is still "mysql" rather than "mariadb".

I'm just gonna trust you that this works and isn't a deprecated value!

Reply:
Confusing!